src/aap_eda/analytics/collector.py

```diff
@@ -35,7 +35,10 @@ def _package_class() -> package.Package:
         return package.Package
 
     def _is_shipping_configured(self) -> bool:
-        if not application_settings.INSIGHTS_TRACKING_STATE:
+        if (
+            not flag_enabled("FEATURE_EDA_ANALYTICS_ENABLED")
+            or not application_settings.INSIGHTS_TRACKING_STATE
```

Comment:
`INSIGHTS_TRACKING_STATE` needs to be exposed in the credential. The users need to restart the service to toggle the flag, but they can easily turn on/off analytics through the credential.

Reply:
Added